openreview/arr/process/change_reviewer_nomination_process.py

```diff
@@ -0,0 +1,10 @@
+def process(client, edit, invitation):
+    # Get paper number
+    note = edit.note
+    forum = client.get_note(note.id)
+    volunteers = note.content.get('reviewing_volunteers', {}).get('value', [])
+    authorids = forum.content.get('authorids').get('value')
+
+    for v in volunteers:
+        if v not in authorids:
+            raise openreview.OpenReviewException(f'Volunteer {v} is not an author of this submission')
```

Comment:
what should we do if the exception is raised?

Reply:
The naming of the file is incorrect, it is named process but it's actually a preprocess - the error would just appear before the form gets submitted